### ISSUE TITLE
Extra text displayed if toctree with a caption is diplayed on current page

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/t3SphinxThemeRtd#130]
```
**Table of Contents**

QUICK LINKS

.. toctree::
   :caption: Quick Links
   :titlesonly:

   ApiOverview/Database/Index
   ApiOverview/DependencyInjection/Index
   ApiOverview/Fal/Index

```

![toctree](https://user-images.githubusercontent.com/13206455/67591395-9d06d280-f75d-11e9-82a2-58905c0b60c1.png)

I think the "Figure: Quick Links" should not be displayed.

**Comment by sypets** [comment]
[sass/_t3_addons.sass](https://github.com/TYPO3-Documentation/t3SphinxThemeRtd/blob/master/sass/_t3_addons.sass):

```
 .rst-content
155   p.caption
156     &::before
157       content: "Figure: "
158     font-size: 14px
159     line-height: 18px

```

The code adds "Figure" to p.caption. If this were changed to only apply to images, it should be ok.

**Comment by marble** [comment]
I'll take care.

**Comment by sypets** [comment]
Is already fixed, see #131.
